### ISSUE TITLE
file extension for a json-ld document should be .jsonld

### ISSUE THREAD

**Issue by melvincarvalho** [w3c-ccg/did-method-web#8]
> For the domain name `w3c-ccg.github.io`, the `did.json` will be available 

The extension in the the current version of the did:web spec uses `.json` in a number of places

But the official extension for a json-ld document is `.jsonld`

Is this worth changing?

**Comment by pmcb55** [comment]
Yeah, interesting - I brought up this very question today with @dmitrizagidulin (only now am I looking at the open issues in this repo!). Personally I agree, and I think it is worth stipulating the official extension of `.jsonld`.
